ci/restore_questionnaire_state.yaml

```diff
@@ -17,6 +17,17 @@ run:
   args:
     - -exc
     - |
+      VARS=('PROJECT_ID' 'BUCKET_NAME' 'BACKUP_NAME' 'FILE_PREFIX')
+
+      for VAR in "${VARS[@]}"
+      do
+        if [ -z "${!VAR}" ]
+        then
+          echo "$VAR variable must be set"
+          exit 1
+        fi
+      done
```

Comment:
Minor, consider using bash expansion or reduce the num lines but have `do` and `then` on the same line.

```bash
PARAMS=('PROJECT_ID' 'BUCKET_NAME' 'BACKUP_NAME' 'FILE_PREFIX')
for param in "${PARAMS[@]}"; do
  : "${!param:?\"Error: $param must be set\"}"
done
``

Reply:
Added, thanks.